CONTRIBUTION.md

```diff
@@ -0,0 +1,31 @@
+Contributing
+============
+
+It's good to hear that you want to contribute to Pony!
+
+First of all please [search existing issues][complete-issue-list]. Apart from bug reports you will
+find feature requests and work currently in progress too. If you cannot find a suitable issue —
+[create a new one][new-issue]. Potential pull requests are no exception. Before you start working,
+lets have a discussion and agree on the solution and scope.
+
+Each of [bug report](#bug-report), [feature request](#feature-request) or [pull request](#pull-request)
+has to cover different aspects described below.
+
+Bug report
+----------
+Provide a code example together with expected and actual results. If possible, try to isolate the
+problem and provide just enough code to demonstrate it. Add any related information which might help
+to fix the issue.
+
+Feature request
+---------------
+Define the context and the problem a feature solves. List potential use cases and give some code
+examples.
+
+Pull request
+------------
+Provide the same information as you would for a [feature request](#feature-request). Also briefly
+describe how the implementation might look like.
+

```

Comment:
I Think coding standards are very important and maybe should be built into the compiler like go does(a little bit). Also someone should write a tool like gofmt which will take pony code and format it. It makes formatting things a lot easier. When I was in the java world we always had our own custom formatting files for a project targeting different IDE's and it was a pain to manage. So it would be great if there was a single standard which we can all follow.